components/channel_notifications_modal/channel_notifications_modal.jsx

```diff
@@ -64,7 +73,7 @@ export default class ChannelNotificationsModal extends React.Component {
             user_id: currentUserId
         };
 
-        updateChannelNotifyProps(data, options,
+        this.props.actions.updateChannelNotifyProps(data, options,
```

Comment:
After making the change for my other comment, you'll need to modify usage of this action inside the component a little bit. Instead of passing in callbacks, you'll need to `await` on it instead.

So instead of 
```
 this.props.actions.updateChannelNotifyProps(data, options, successCallback, errorCallback);
```
you would do (notice the parameters are slightly different as well)
```
const {err} = await this.props.actions.updateChannelNotifyProps(userId, channelId, options);
if (err) {
  // handle error case
} else {
  // handle success case
}
```

You would also need to make the function this is all being done in an `async` function